src/croniter/croniter.py

```diff
@@ -830,9 +833,18 @@ def value_alias(cls, val, field_index, len_expressions=UNIX_CRON_LEN):
             # do not support 0 as a month either for classical 5 fields cron,
             # 6fields second repeat form or 7 fields year form
             # but still let conversion happen if day field is shifted
-            (field_index in [DAY_FIELD, MONTH_FIELD] and len_expressions == UNIX_CRON_LEN) or
```

Comment:
this one is hardier to read now :(

Reply:
Yeah I agree :(

If you're OK with merging slightly harder to read for the sake of getting black applied, I can come back to these and clean them up.

If you would rather I fix the hard to read ones now let me know and I can clean them up by moving them into named variables to make the conditions easier to read.

Reply:
just using `fmt: off` for that particular line should do it, doesnt it ? it should stay as-is

Reply:
Yes I can do that as well, but I think it's best practice to try and avoid using too much `# fmt: off` when possible, since it can mean formatting inconsistencies can leak in.

Let me remove this particular change from this PR and we can merge this one and i'll address this separate

Reply:
Yes ok

Reply:
Done